rsciio/bruker/_api.py

```diff
@@ -1290,6 +1289,41 @@ def py_parse_hypermap(virtual_file, shape, dtype, downsample=1):
 
 
 def file_reader(filename, *args, **kwds):
+    """
+    Reads a bruker ``bcf`` or ``spx`` file
+
+    Parameters
+    ----------
+    select_type : ``'spectrum_image'``, ``'image'`` or None
+        If specified, only the corresponding type of data, either spectrum
+        image or image, is returned.
+        By default (None), all data are loaded.
+    index : int, None or str
+        Allow to select the index of the dataset in the ``bcf`` file, which
+        can contains several datasets. Default ``None`` value result in loading
+        the first dataset. When set to ``'all'``, all available datasets
+        will be loaded and returned as separate signals.
+    downsample : int
+        The downsample ratio of hyperspectral array (height and width only),
+        can be integer >=1, where ``1`` results in no downsampling (default 1).
+        The underlying method of downsampling is unchangeable: sum. Differently
+        than :py:func:``skimage.measure.block_reduce``, it is memory efficient
+        as it doesn't create intermediate arrays
+    cutoff_at_kV : int, float, ``'zealous'``, ``'auto'`` or None
+        It can be used either to crop or enlarge energy (or number of channels)
+        range at max values. It can be used to conserve memory or enlarge
+        the range if needed to mach the size of other file. Default value is None
+        (which does not influence size). Numerical values should be in kV.
+        ``'zealous'`` truncates to the last non zero channel (this option
+        should not be used for stacks, as low beam current EDS can have different
+        last non zero channel per slice). ``'auto'`` truncates channels to SEM/TEM
+        acceleration voltage or energy at last channel, depending which is smaller.
+        In case the hv info is not there or hv is off (0 kV) then it fallbacks to
+        full channel range.
```

Comment:
```suggestion
        acceleration voltage (hv) or energy at last channel, depending which is smaller.
        In case the hv info is not there or hv is off (0 kV) then it falls back to
        the full channel range.
```